specifyweb/frontend/js_src/lib/components/SchemaConfig/Components.tsx

```diff
@@ -33,7 +33,9 @@ export function SchemaConfigHeader({
         {schemaText.changeBaseTable()}
       </Button.Small>
       <span className="-ml-2 flex-1" />
-      <Button.Save onClick={handleSave}>{commonText.save()} </Button.Save>
+      <Button.Save className="!py-1 !px-2" onClick={handleSave}>
```

Comment:
noo, we talked about this. don't duplicate styles like that - if we change them (and we do from time to time), this place would remain unchanged.

Remember how you could use Button.Small and variant={} instead?